aequilibrae/project/network/node.py

```diff
@@ -91,8 +91,6 @@ def renumber(self, new_id: int):
 
         curr.execute("BEGIN;")
         curr.execute("Update Nodes set node_id=? where node_id=?", [new_id, self.node_id])
-        curr.execute("Update Links set a_node=? where a_node=?", [new_id, self.node_id])
```

Comment:
this doesn't look like documentation change?

Reply:
Not indeed.  But realized that this was covered by a trigger